apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/components/CareerTimelineAndRecruitment.tsx

```diff
@@ -137,8 +137,8 @@ const CareerTimelineAndRecruitment = ({
               <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "This section is similar to your traditional resume and describes your experiences across work, school, and life. You’ll be able to reuse this information on each application you submit on the platform, speeding up the process and ensuring that your information is always up-to-date.",
-                  id: "xXEhNq",
```

Comment:
I don't think you need to delete the "You'll be able to..." sentence

Reply:
That does make sense. Added back in [add back last sentence](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9011/commits/1f0ce7b72a42f3a7544280c34e77c4154a22ff98)